edgelet/docker-rs/src/apis/container_api.rs

```diff
@@ -196,11 +200,13 @@ impl<C: hyper::client::Connect> ContainerApi for ContainerApiClient<C> {
         // if let Err(e) = uri {
         //     return Box::new(futures::future::err(e));
         // }
-        let mut req = hyper::Request::new(method, uri.unwrap());
+        let mut req = hyper::Request::new(hyper::Body::empty());
+        *req.method_mut() = method;
+        *req.uri_mut() = uri.unwrap();
```

Comment:
It does, but it doesn't support typed headers (since `typed-headers` only adds functions to the `HeaderMap`, not the `Builder`, so I'll still have to mutate `headers_mut()` after building. Will do.